From e4d3b07e31a08591b6b1c7d4b43bc3ff61bceca4 Mon Sep 17 00:00:00 2001
From: Chen Meng J <meng.j.chen@intel.com>
Date: Mon, 13 Jan 2020 14:16:38 +0800
Subject: [PATCH] media: intel-ipu4: pdata for mcp4725

Message for Open Source:
pdata for mcp4725.

Message for Internal:
pdata for mcp4725.

Change-Id: Ic0df9f3bd389563e083645545f173f87ac9b6363
Signed-off-by: Chen Meng J <meng.j.chen@intel.com>
---
 drivers/media/platform/intel/ipu4-bxt-p-pdata.c | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/drivers/media/platform/intel/ipu4-bxt-p-pdata.c b/drivers/media/platform/intel/ipu4-bxt-p-pdata.c
index 703763f..93a3810 100644
--- a/drivers/media/platform/intel/ipu4-bxt-p-pdata.c
+++ b/drivers/media/platform/intel/ipu4-bxt-p-pdata.c
@@ -224,6 +224,15 @@
 
 #ifdef CONFIG_INTEL_IPU4_IMX274
 
+static struct ipu_isys_subdev_info vcm_mcp4725_common_sd = {
+	.i2c = {
+		.board_info = {
+			I2C_BOARD_INFO("vcm-mcp4725", 0x60),
+		},
+		.i2c_adapter_id = 2,
+	}
+};
+
 #define IMX274_LANES		4
 #define IMX274_I2C_ADDRESS	0x1a
 
@@ -1655,6 +1664,7 @@
 #ifdef CONFIG_INTEL_IPU4_IMX274
 		&imx274_crl_sd,
 		&imx274_b_crl_sd,
+		&vcm_mcp4725_common_sd,
 #endif
 #ifdef CONFIG_INTEL_IPU4_IMX290
 		&imx290_crl_sd,
-- 
1.9.1

